CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+Version 0.3.1 *(08/02/2017)*
```

Comment:
To be honest we could remove the date completely and add links to the GH release instead. What you think @ataulm?